pdtf-api-1.2.0.yaml

```diff
@@ -0,0 +1,158 @@
+openapi: 3.0.3
+info:
+  title: Property Data Trust Framework API
+  version: 1.2.0
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+  description: This API defines how participants in the UK Property Data Trust Framework can securely share property data, together with the verifiable provenance of that data. The API is built around a variation of OpenID Connect Verified Claims schema.
+servers:
+  - url: http://api.pdtf-example.com
+    description: Example PDTF data source
+tags:
+  - name: transactions
+    description: Transactions API
+paths:
+  /transactions:
+    post:
+      tags:
+        - transactions
+      summary: Add claims on a transaction
+      operationId: addTransactionClaims
+      responses:
+        "201":
+          description: Created
+        "400":
+          description: Bad Request - the supplied claims contain one or more validation errors
+        "401":
+          description: Unauthorized
+      description: Add claims on a transaction
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: https://raw.githubusercontent.com/Property-Data-Trust-Framework/schemas/master/src/schemas/pdtf-transaction.json
```

Comment:
I don't think any of these schema refs should be the `pdtf-transaction.json` schema - the framework only allows data to be supplied wrapped in a verified claim, so these would all be `https://raw.githubusercontent.com/Property-Data-Trust-Framework/schemas/master/src/schemas/pdtf-verified-claims.json`

Reply:
Makes sense, I was unsure about this one. Parts of the "transaction" schema would go in the `claims` block?